src/utils/sortGoods.ts

```diff
@@ -0,0 +1,28 @@
+import { SortType } from '../types/SortType';
+
+export function sortGoods(
+  goods: string[],
+  sortField: SortType = SortType.Default,
```

Comment:
The `SortType.Default` enum value should be renamed to `SortType.DEFAULT` to follow the convention of using uppercase for enum values, indicating that they are constants.